src/ol/render/canvas/replay.js

```diff
@@ -1202,31 +1202,43 @@ ol.inherits(ol.render.canvas.PolygonReplay, ol.render.canvas.Replay);
  */
 ol.render.canvas.PolygonReplay.prototype.drawFlatCoordinatess_ = function(flatCoordinates, offset, ends, stride) {
   var state = this.state_;
+  var fill = state.fillStyle !== undefined;
+  var stroke = state.strokeStyle != undefined;
+  if (!fill && !stroke) {
+    return offset;
```

Comment:
Can you add a test for this? I fear we might need to return something else, right.